packages/react/src/field/Field.tsx

```diff
@@ -77,10 +77,11 @@ export const Field = forwardRef<HTMLDivElement, FieldProps>(
                   <Button
                     aria-label="Information hover"
                     border="0"
-                    h="12"
+                    color="fg.secondary"
+                    h="16"
                     icon={<IconCircleQuestion />}
                     p="0"
-                    w="12"
+                    w="20"
```

Comment:
icon buttons need to be square - let's change this to 16 and use `size` (remove the extra `h`) so it's clear that the intent is for this to be a square
```suggestion
                    size="16"
```

Reply:
Okay vai